src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -0,0 +1,583 @@
+                    {{1}, {ov::element::i8}},
+                }),
+            PrecisionsRestriction::create<ov::opset1::ConvolutionBackpropData>({
+                    {{0}, {ov::element::u8, ov::element::i8}},
+                    {{1}, {ov::element::i8}}
+                }),
+            PrecisionsRestriction::create<ov::opset1::GroupConvolution>({
+                    {{0}, input0LowPrecisionList},
+                    {{1}, {ov::element::i8}}
+                }),
+            PrecisionsRestriction::create<ov::opset1::Multiply>({
+                    {{0}, {ov::element::u8}},
+                    {{1}, {ov::element::i8}},
+                }),
+            PrecisionsRestriction::create<ov::opset1::MatMul>({
+                    {{0}, {ov::element::u8, ov::element::i8}},
+                    {{1}, {ov::element::i8}}
+                }),
+            PrecisionsRestriction::create<ov::opset5::LSTMSequence>({
+                    {{0, 1}, {ov::element::u8, ov::element::i8}},
+                }),
+            PrecisionsRestriction::create<ov::opset6::GRUSequence>({
+                    {{0, 1}, {ov::element::u8, ov::element::i8}},
+                }),
+        });
+
+    auto quantizationRestrictions = std::vector<QuantizationGranularityRestriction>({
+            QuantizationGranularityRestriction::create<ov::opset1::Convolution>({0}),
+            QuantizationGranularityRestriction::create<ov::opset1::ConvolutionBackpropData>({0})
+        });
+
+    // for GNA networks reference execution
+    bool updatePrecision = true;
+    if (hasINT16orINT32Levels) {
+        updatePrecision = false;
+        supportedPrecisions = std::vector<PrecisionsRestriction>({});
+    }
+
+    ov::pass::Manager lptManager;
+    lptManager.register_pass<ngraph::pass::low_precision::LowPrecision>(
+        supportedPrecisions,
+        quantizationRestrictions,
+        LayerTransformation::Params(updatePrecision, ov::element::f32, defaultPrecisions));
+    lptManager.get_pass_config()->set_callback<ngraph::pass::low_precision::MarkupPrecisions>([](const_node_ptr& node) -> bool {
+        if (const auto mulitply = std::dynamic_pointer_cast<const ov::opset1::Multiply>(node)) {
+            return !MultiplyToGroupConvolutionTransformation::canBeTransformedToGroupConvolution(mulitply);
+        }
+        return false;
+    });
+    lptManager.get_pass_config()->set_callback<ngraph::pass::low_precision::ConvolutionBackpropDataTransformation>(
+        [&defaultPrecisions](const_node_ptr& node) -> bool {
+            return LayerTransformation::isAsymmetricQuantization(node, defaultPrecisions) ||
+                WeightableLayerTransformation::isAsymmetricOnWeights(node, defaultPrecisions);
+        });
+    lptManager.get_pass_config()->set_callback<ngraph::pass::low_precision::MultiplyToGroupConvolutionTransformation>([](const_node_ptr& node) -> bool {
+        return true;//MultiplyToGroupConvolutionTransformation::isDynamicOrScalar(node);
+    });
```

Comment:
This callback is an equivalent to transformation disabling. Since we are moving the code to another place, can we explicitly disable the transformation to make the code clearer?

Reply:
Updated